package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "sactuary-date-poc",
+  "version": "0.1.0",
+  "dependencies": {
+    "date-fp": "*",
+    "lodash.curry": "^3.0.2",
```

Comment:
I don't think so, I just copy and pasted the `package.json` from the original `date-fp` where it's used.